siliconcompiler/flows/showflow.py

```diff
@@ -29,16 +31,22 @@ def setup(chip, flowname='showflow', filetype=None, screenshot=False, np=1):
     if not filetype:
         raise ValueError('filetype is a required argument')
 
-    flow.node(flowname, 'import', 'builtin', 'import')
+    flow.node(flowname, 'import', siliconcompiler, 'import')
 
+    if filetype not in chip.getkeys('option', 'showtool'):
+        raise SiliconCompilerError(f'Show tool for {filetype} is not defined.')
     show_tool = chip.get('option', 'showtool', filetype)
 
+    show_tool_module = chip._lookup_toolmodule_by_name(show_tool)
+    if not show_tool_module:
+        raise SiliconCompilerError(f'Cannot determine tool module for {show_tool}.')
```

Comment:
I think one pitfall is this requires the chip object already has a flow set up that includes a node using the show tool. Maybe fall back to doing a target-style search of `[name, f'siliconcompiler.tools.{name}', f'siliconcompiler.tools.{name}.{name}']`?

Reply:
Although we might need to invert the priority on that search. Otherwise, if you happen to have the KLayout standalone package installed, "klayout" would match on that rather than the SC tool driver.